content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,293 @@
+| **Pod CIDR (--pod_cidr)**|Enter the CIDR pool that will be used to assign IP addresses to pods in the cluster. The pod IP addresses should be unique and should not overlap with any virtual machine IPs in the environment.|
+| **Service IP Range (--svc_ip_range)**|Enter the IP address range that will be used to assign IP addresses to services in Kubernetes clusters. The service IP addresses should be unique and not overlap with any virtual machine IPs in the environment.|
+
+<br />
+
+
+#### MAAS Account Information
+
+<WarningBox>
+
+When specifying the MAAS endpoint, do not add a trailing `/` as it can result in DNS resolution errors.
+
+</WarningBox>
+
+|**Parameter**| **Description**|
+|-------------|----------------|
+| **API Endpoint** |Enter the MAAS API endpoint without a trailing /. This can be a domain or IP address. Example: ``http://10.11.12.13:5240/MAAS.``|
+| **API Key** |Enter an active MAAS API key to use for authentication.|
+
+<br />
+
+7. When the installer prompts you, select the following to configure the MAAS server:
+
+    - Availability Zone
+    - Domain
+    - Resource Pool
+    - One node (no HA) or three nodes (HA)
+
+    <br />
+
+    The installer generates a ``pcg.yaml`` configuration file and displays the file location on the console. For example: <br />
+
+``Config created:/opt/spectrocloud//User-defined-MaaS-Gateway-Name-20210805155034/pcg.yaml``
+
+<br />
+
+8. Copy the ``pcg.yaml`` file out of ``/tmp/install-user-defined-MaaS-Gateway_Name-20210805155034/pcg.yaml`` and into ``/tmp``.
+
+``` bash
+cp /tmp/install-user-defined-MaaS-Gateway_Name-20210805155034/pcg.yaml /tmp
+```
+<br />
+
+<WarningBox>
+
+After you copy ``pcg.yaml``, delete it from ``/tmp/install-user-defined-MaaS-Gateway_Name/pcg.yaml`` because it contains your Spectro Cloud credential.
+
+</WarningBox>
+
+<br />
+
+9. To deploy the gateway, copy the following code snippet to your terminal and provide the gateway configuration file as input.
+<br />
+
+    ```bash
+    docker run -it --rm \
+    --net=host \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/opt/spectrocloud \
+    gcr.io/spectro-images-public/release/spectro-installer:1.0.12 \
+    -s true \
+    -c //opt/spectrocloud/pcg.yaml
+    ```
+The installer selects available bare metal machines in your MAAS environment on which to install the gateway. If the deployment fails due to misconfiguration, update the gateway configuration file and rerun the command.
+
+If you need assisstance, you can send a message to our **#support-sustaining** slack channel with questions.
+
+
+# Validation
+
+Once installed, the gateway registers itself with Palette's SaaS portal. To verify the gateway is registered, navigate to **Tenant Settings > Private Cloud Gateways > MAAS** and verify the gateway is listed on the Manage Private Cloud Gateways page. 
```

Comment:
```suggestion
Once installed, the gateway registers itself with Palette. To verify the gateway is registered, navigate to **Tenant Settings > Private Cloud Gateways > MAAS** and verify the gateway is listed on the Manage Private Cloud Gateways page. 
```
The portal is the UI, it's the API server that actually accepts the registration so we can just say Palette.